src/implementations/vanilla/components/page-header-standardised/page-header-standardised-print.scss

```diff
@@ -12,13 +12,29 @@
 }
 
 .ecl-page-header-standardised__meta {
-  color: map.get(theme.$color, 'grey');
+  color: map.get(theme.$color, 'grey-100');
   font: map.get(theme.$font-print, 'm');
   font-family: map.get(theme.$font-family-print, 'alt');
```

Comment:
Review fonts for print version (meta & h1). Existing FRONT-2210 ticket to fix this on all component. So either correct it here directly here or wait for the general ticket?